internals/daemon/daemon_test.go

```diff
@@ -98,6 +98,29 @@ func (h *fakeHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	h.lastMethod = r.Method
 }
 
+func (s *daemonSuite) TestAddCommand(c *C) {
+	const endpoint = "/v1/addedendpoint"
+	var handler fakeHandler
+	getCallback := func(c *Command, r *http.Request, s *UserState) Response {
+		handler.cmd = c
+		return &handler
+	}
+	command := Command{
+		Path:    endpoint,
+		GuestOK: true,
+		GET:     getCallback,
+	}
+	API = append(API, &command)
```

Comment:
This needs to be removed in a defer so this is not polluting the environment for other tests.